tests/test_core.py

```diff
@@ -4,11 +4,31 @@
 import numpy as np
 import pytest
 
-from albumentations.core.transforms_interface import to_tuple, ImageOnlyTransform, DualTransform
 from albumentations.augmentations.bbox_utils import check_bboxes
-from albumentations.core.composition import OneOrOther, Compose, OneOf, PerChannel
-from albumentations.augmentations.transforms import HorizontalFlip, Rotate, Blur, MedianBlur
-from .compat import mock, MagicMock, Mock, call
+from albumentations.augmentations.transforms import (
+    Blur,
+    HorizontalFlip,
+    MedianBlur,
+    Rotate,
+)
+from albumentations.core.composition import (
+    Compose,
+    OneOf,
+    OneOrOther,
+    PerChannel,
+)
+from albumentations.core.transforms_interface import (
+    DualTransform,
+    ImageOnlyTransform,
+    to_tuple,
+)
+
+from .compat import (
+    MagicMock,
+    Mock,
+    call,
+    mock,
+)
```

Comment:
I'm not really sure that I like this style more then plain lines. it's 22 lines of code instead of 3